.circleci/main.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - run:
           no_output_timeout: 20m
           command: |-
-            GOMAXPROCS=8 KUBECONFIG=/dev/null PACH_CONFIG=/dev/null \
+            GOMAXPROCS=8 CGO_ENABLED=0 KUBECONFIG=/dev/null PACH_CONFIG=/dev/null \
```

Comment:
@acohen4 I wanted to make sure this was okay as well. I think from a testing perspective it works fine, however I don't know if we actually do compile anything that relies on `cgo`. We have a file that inits dex only if cgo is excluded